src/Modal/ModalActions.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from '../Button';
+import styles from "./Modal.module.scss";
+
+const ModalActions = ({
+  onConfirmClick,
+  confirmLabel,
+  closeModal,
+  cancelLabel,
+}) => (
+  <div className={styles.actions}>
+    <Button
+      border
+      className={styles.btnCancel}
+      accent="negative"
+      onClick={closeModal}
+    >
+      {cancelLabel}
+    </Button>
+    {onConfirmClick && (
+      <Button
+        variant="fill"
+        accent="positive"
+        onClick={e => onConfirmClick(e, closeModal)}
```

Comment:
`e => onConfirmClick(e, closeModal)` just why?
Class property in `Modal`
```jsx
  onConfirmClick = () => {
     this.closeModal();
     this.props.onConfirmClick();
  }
```
and pass it down to `ModalActions`

Reply:
Reasonable argument. Still no need for creating closure on every render.